src/app/widgets/cat-source/cat-source-tev/cat-source-tev.component.html

```diff
@@ -1,28 +1,105 @@
 <div *ngIf="(catalog)">
   <div class="container">
     <div class="col-md-6">
-      <h3>{{getSource().common_name}}</h3>
+      <h2>{{d.common_name}}</h2><br>
 
-      <h5>Basic info</h5>
+      <h4><b>Basic Info</b></h4>
       <ul>
-        <li>Common name: {{getSource().common_name}}</li>
+        <li>Common name: {{d.common_name}}</li>
+        <li>Gamma names: {{source.gamma_names_str()}}</li>
+        <li>Fermi names: {{source.fermi_names_str()}}</li>
+        <li>Other names: {{source.other_names_str()}}</li>
+        <!-- TODO: Combine the three above into one "Other names"? -->
+        <li>Location: {{d.where}}</li>
+        <li>Class: {{source.class_str()}}</li>
+        <br>
+        <li>TeVCat ID: {{d.tevcat_id}}</li>
+        <li>TevCat2 ID: {{d.tevcat2_id}}</li>
+        <li>TeVCat name: {{d.tevcat_name}}</li>
```

Comment:
This is what you have for TeVCat:
![screen shot 2017-07-13 at 13 30 58](https://user-images.githubusercontent.com/852409/28164397-934a0426-67cf-11e7-99ef-00770e97b8e8.png)

I think it would be nice if you generated links one can click, which open up a new browser tab.

The TeVCat info could possibly be concatenated into one bullet, like this:
```
* On TeVCat: TeV J0006+729 (TeVCat ID: 227 ,  TevCat2 ID: rC5JCj)
```
and have the "TeVCat ID: 227" and "TevCat2 ID: rC5JCj" be clickable links